Makefile

```diff
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=/usr
```

Comment:
/usr/local is the default prefix for any source project. if you want the prefix to be /usr, then pass `make prefix=/usr` to the build (and also to `make prefix=/usr install`

Reply:
that didn't work for me though

Reply:
then you probably did `prefix=/usr make` - that won't work, you need to pass it to make after make. you can alternatively put `prefix=/usr` into config.mak which is included by the build when you make the debian build description. but don't add config.mak to the repo.

Reply:
i put it in the debian rules file